tests/matlab_test_data_collectors/python_testers/expandMatrix_test.py

```diff
@@ -13,18 +13,19 @@ def test_expand_matrix_test():
     num_collected_values = len(os.listdir(collected_values_folder))
     for i in range(num_collected_values):
         filepath = os.path.join(collected_values_folder, f'{i:06d}.mat')
-        recorded_data = loadmat(filepath)
+        recorded_data = loadmat(filepath, simplify_cells=True)
 
         matrix = recorded_data['matrix']
         matrix = np.squeeze(matrix)
         input_args = recorded_data['input_args']
-        exp_coeff = input_args[0][0]
-        if len(input_args[0]) == 2:
-            edge_val = float(input_args[0][1])
+        exp_coeff = input_args
+        if isinstance(input_args[0], np.ndarray):
+            edge_val = float(input_args[1])
+            exp_coeff = input_args[0]
         else:
             edge_val = None
+        exp_coeff = exp_coeff.astype(int)
```

Comment:
why always int?

Reply:
Removed the cast. Although we have another cast inside of the `expandMatrix` function:
```python
exp_coeff = np.array(exp_coeff).astype(int).squeeze()
```